### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by rdimitrov**

### CODE REVIEW

**Comment by rdimitrov** [comment]
Hey

**Comment by rdimitrov** [comment]
Test

File: README.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ Run it with -
 ```
 curl -d "OpenFaaS Cloud is awesome!" https://rdimitrov.o6s.io/helloworld
 ```
-And to try the QR test, use - 
+And to try the QR test, use[todo] -
```
Comment (nickolaev) [inline README.md:8]: You should reconsider this.

**Pull request closed without merging**